lib/vagrant.pm

```diff
@@ -0,0 +1,34 @@
+package vagrant;
+use testapi;
+use strict;
+use warnings;
+use utils;
+
+our @ISA    = qw(Exporter);
+our @EXPORT = qw(setup_vagrant);
+
+# - install vagrant and all additional packages
+# - launch the required daemons
+#
+# Named parameters:
+# - install_virtualbox: bool
+#   whether to install & setup virtualbox instead of libvirt
+sub setup_vagrant {
+    my ($args) = @_;
+
+    select_console('root-console');
+
+    my $extra_pkgs  = 'vagrant-libvirt';
+    my $daemon_name = 'libvirtd';
+    my $user_group  = 'vboxusers';
+
+    if ($args->{install_virtualbox}) {
+        $extra_pkgs  = 'virtualbox';
+        $daemon_name = 'vboxdrv vboxautostart';
+        $user_group  = 'vboxusers';
+    }
+
+    zypper_call('in vagrant $extra_pkgs');
```

Comment:
I think single quotes do not allow `$var` expansion. Same for the other lines.

Reply:
Yes, I just saw that too and pushed a (hopefully) fixed version: https://openqa.opensuse.org/tests/1007256